packages/rke2-cilium/generated-changes/patch/values.yaml.patch

```diff
@@ -6,16 +6,27 @@
    override: ~
 -  repository: "quay.io/cilium/cilium"
 +  repository: "rancher/mirrored-cilium-cilium"
```

Comment:
Do we need to prefix this with `"{{ template "system_default_registry" . }}"` ? Are there any other places where we have a repository and tag reference, instead of an image, that we might not be prefixing the default registry at the moment?

Reply:
I think we are currently well covered. By grepping on `cilium.image` on the templates directory, I can see all processing the template:
```
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
cilium-agent/daemonset.yaml:        image: "{{ template "system_default_registry" . }}{{ include "cilium.image" .Values.image }}"
```

As far as I know, it is not possible to add templates in the values.yaml :(. That would be really helpful. In 1.12.0, cilium-agent daemonset added a new initContainer, we missed it and thus we got an angry user.